graph_neural_network/dist_train_rgnn.py

```diff
@@ -412,10 +423,9 @@ def run_training_proc(local_proc_rank, num_nodes, node_rank, num_training_procs,
     if args.split_training_sampling:
       assert(not args.num_training_procs > torch.cuda.device_count() // 2)
   
+  world_size = args.num_nodes * args.num_training_procs
   if args.node_rank == 0:
-    world_size = args.num_nodes * args.num_training_procs
     submission_info(mllogger, 'GNN', 'reference_implementation')
```

Comment:
We might want to also change the name here from `'GNN'` to `mllog_constants.GNN` now that this constant is included in the latest mllog constants.